tests/ut_common_copytotarget.py

```diff
@@ -0,0 +1,27 @@
+"""
+The unit test check if copy_to_target(file_expr, target) works fine.
+
+"""
+import sys
+import os
+import shutil
+import tempfile
+import build_component
+
+def main():
+    # create a new temporary file and a new temporary directory.
+    temp = tempfile.NamedTemporaryFile()
```

Comment:
`temp` is a pretty nondescriptive variable name. How about `temporary_file`? (Length shouldn't matter much, because we are using the identifier only three times overall). `temporary_dir` would be a logical extension, then.